rectools/models/nn/transformers/bert4rec.py

```diff
@@ -301,6 +304,10 @@ class BERT4RecModel(TransformerModelBase[BERT4RecModelConfig]):
         When set to ``None``, "cuda" will be used if it is available, "cpu" otherwise.
         If you want to change this parameter after model is initialized,
         you can manually assign new value to model `recommend_torch_device` attribute.
+    get_val_mask_func_kwargs: optional(InitKwargs), default ``None``
```

Comment:
We need to write everywhere in docstrings "Make sure all dict values have JSON serializable types."
Otherwise we will fail to serialize the model